viewlet/tests/test_viewlet.py

```diff
@@ -16,10 +15,15 @@
 from ..loaders import jinja2_loader
 from ..loaders.jinja2_loader import get_env
 
-if django.VERSION >= (1, 7):
-    django.setup()
+from ..compat import Context as ComptContext
+
+if django.VERSION[:2] < (1, 8):
+    from django.template.loader import get_template_from_string
+else:
+    from django.template import engines
```

Comment:
It would be great to move it to `compat`, so that we could reduce the number of version checks in the main code.
Also, we don't need sublisting, this will also work:
```py
django.VERSION < (1, 8)
```